rules-xml/api/src/main/resources/CatalogManager.properties

```diff
@@ -0,0 +1,8 @@
+#CatalogManager.properties
+verbosity=1
+relative-catalogs=false
+# Always use semicolons in this list
+catalogs=./core-catalog.xml
+prefer=public
+static-catalog=yes
+allow-oasis-xml-catalog-pi=yes
```

Comment:
Yes, I think we will need some Java EE ones as well, but this is a good start. +1